### PR TITLE
♻️Fixed Unit test director v2

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- fixes tests in dv-2
the fix for the clusters is ugly, but these will disappear soon.

The issue arise because the ClusterGet is the same as the internal model and the webserver likes to have them frozen (after validators cannot run in frozen models).
since this is pretty much unused I will not waste any more time on this.

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/models-library/src/models_library/clusters.py [hunk 1]
```diff
@@ -222,29 +221,22 @@ class Cluster(BaseCluster):
...
-            CLUSTER_USER_RIGHTS.model_dump()
-            if is_default_cluster
-            else CLUSTER_ADMIN_RIGHTS.model_dump()
+            CLUSTER_USER_RIGHTS if is_default_cluster else CLUSTER_ADMIN_RIGHTS
         ):
             msg = f"the cluster owner access rights are incorrectly set: {access_rights[owner_gid]}"
             raise ValueError(msg)
-        values["access_rights"] = access_rights
-        return values
+        object.__setattr__(self, "access_rights", access_rights)
```
Comment (pcrespov) [inline packages/models-library/src/models_library/clusters.py:241]: Add a reference to this trick to overcome "frozeness" this way we have a context of why this was done this way?

File: services/director-v2/src/simcore_service_director_v2/models/comp_runs.py [hunk 2]
```diff
@@ -140,10 +144,10 @@ def convert_null_to_empty_metadata(cls, v):
                     "cluster_id": 123,
                     "iteration": 12,
                     "result": "SUCCESS",
-                    "created": "2021-03-01 13:07:34.19161",
-                    "modified": "2021-03-01 13:07:34.19161",
-                    "started": "2021-03-01 8:07:34.19161",
-                    "ended": "2021-03-01 13:07:34.10",
+                    "created": "2021-03-01T13:07:34.191610",
```
Comment (pcrespov) [inline services/director-v2/src/simcore_service_director_v2/models/comp_runs.py:147]: Q: do these columns are defined in the database as `sa.DateTime(timezone=True)`  i.e. with timezone?
SEE https://github.com/ITISFoundation/osparc-simcore/issues/3797

Comment (sanderegg) [reply services/director-v2/src/simcore_service_director_v2/models/comp_runs.py:147]: no. and the problem there was I copy/pasted from adminer the date time. and this is not a proper date time format.

File: packages/models-library/src/models_library/clusters.py (hunk 1 above)
Comment (sanderegg) [reply packages/models-library/src/models_library/clusters.py:241]: ok

**Pull request merged**